controllers/src/test/scala/cromwell/pipeline/controller/AkkaHttpClientTest.scala

```diff
@@ -1,30 +1,31 @@
-package cromwell.pipeline.service
+package cromwell.pipeline.controller
 
 import akka.event.LoggingAdapter
 import akka.http.scaladsl.model.{ HttpRequest, HttpResponse }
 import akka.http.scaladsl.settings.ConnectionPoolSettings
 import akka.http.scaladsl.{ HttpExt, HttpsConnectionContext }
+import cromwell.pipeline.service.Response
 import org.mockito.Matchers.any
 import org.mockito.Mockito.when
 import org.scalatest.{ AsyncWordSpec, BeforeAndAfterAll, Matchers }
 import org.scalatestplus.mockito.MockitoSugar
 
 import scala.concurrent.{ ExecutionContext, Future }
 
-class ControllerClassTest extends AsyncWordSpec with Matchers with MockitoSugar with BeforeAndAfterAll {
+class AkkaHttpClientTest extends AsyncWordSpec with Matchers with MockitoSugar with BeforeAndAfterAll {
```

Comment:
BeforeAndAfterAll seems like unused feature

Reply:
Looks like it's true

Reply:
Removed